apps/angular/src/app/features/anime/anime-management/anime-editing/anime-editing.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, DestroyRef, inject } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { ActivatedRoute, Router } from '@angular/router';
+import { AnimeFormService } from '@js-camp/angular/core/services/anime-form.service';
+import { AnimeFormData } from '@js-camp/core/models/anime/anime-form-data';
+import { Observable, map, switchMap, tap } from 'rxjs';
+
+/** Anime editing component. */
+@Component({
+	selector: 'camp-anime-editing',
+	templateUrl: './anime-editing.component.html',
+})
```

Comment:
I've noticed `changeDetection` property is missing in some components.
By default Angular component has `Default` change detection. It might decrease performance.
That's why we prefer to set `changeDetection` to `OnPush` everywhere.
```suggestion
@Component({
	selector: 'camp-anime-editing',
	templateUrl: './anime-editing.component.html',
	changeDetection: ChangeDetectionStrategy.OnPush,
})
```
Please, add this everywhere.

Reply:
By the way, if you create components use Angular CLI you will have this automatically.
Usually we use the following command to create new component:
```bash
npx ng g c anime-editing
```
I'm not sure about NX though, we don't have it in our apps. You may as Pavel Romanov.
It will automatically create required files with correct params. For instance 'camp' prefix will be added automatically as well as 'changeDetection'.

In general it's not recommended to create components/service/pipes/modules/directives automatically. We should generate them using Angular CLI. You can find more details about it in [docs](https://angular.io/cli/generate).